tests/test_servo.py

```diff
@@ -594,6 +612,23 @@ def test_fault_reset(connect_to_slave):
     servo.write("DRV_PROT_USER_OVER_VOLT", data=prev_val, subnode=1)
 
 
+@pytest.mark.ethercat
+def test_fault_reset_eve_xcr(connect_to_slave, get_configuration_from_rack_service):
+    drive_idx, config = get_configuration_from_rack_service
+    drive = config[drive_idx]
+    if drive.identifier != "eve-xcr-e":
+        pytest.skip("The test is only for the EVE-XCR-E")
+    servo, net = connect_to_slave
+    prev_val = servo.read("DRV_PROT_USER_OVER_VOLT", subnode=1)
+    servo.write("DRV_PROT_USER_OVER_VOLT", data=10.0, subnode=1)
+    with SDOReadTimeoutManager(network=net, new_value=net.DEFAULT_ECAT_CONNECTION_TIMEOUT_S * 2):
```

Comment:
Wait a second. Okay, we fixed the test, but the bug will still be present in everest....
What if we moved SDOReadTimeoutManager to source and set it during fault reset?
Let's talk about it together today